pkg/gh/issues.go

```diff
@@ -114,6 +116,17 @@ func (o GitOptions) CommentIssue(ctx context.Context, owner, repo, comment strin
 	return issue.GetHTMLURL(), err
 }
 
+func (o GitOptions) LabelIssue(ctx context.Context, owner, repo string, number int, labels *[]string) error {
```

Comment:
Is there any sense in testing that the labeling is happening when we expect?

Reply:
Are you thinking verifying that the call succeeded here by looking at the returned issue and verifying its state against the arguments, or an end-to-end test to github in the test suite, or both? :)

Reply:
Yeah, good point, maybe an E2E test makes more sense. And maybe not now if we don't have a precedent for it. Treat this as optional — LGTM!